.github/workflows/api-compatibility.yml

```diff
@@ -35,7 +34,7 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@0a12ed9d6a96ab950c8f026ed9f722fe0da7ef32 # v5.0.2
         with:
-          go-version: ~1.21.13
+          go-version: ~1.22.6
```

Comment:
this might be worth its own PR?

Reply:
the PR was built on top of the update to go 1.22, it's ready to review now